tests/tslint.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I wonder if we can just use the same `tslint` and `tsconfig` for both `test` and `src`